napari/layers/points/points.py

```diff
@@ -412,13 +414,13 @@ def __init__(
         self.events.add(
             size=Event,
             current_size=Event,
-            edge_width=Event,
-            current_edge_width=Event,
-            edge_width_is_relative=Event,
+            border_width=Event,
```

Comment:
```suggestion
            border_width=Event,
            edge_width=WarningEmitter(
                trans._(
                    "layer.events.edge_width is deprecated since 0.5.0 and will be removed in 0.6.0. Please use layer.events.border_width ",
                    deferred=True,
                ),
                type_name='interactive',)
```
And respectively all other events, Then ensure that events are emitted (call them in proper laces or connect to a new event).